go.mod

```diff
@@ -46,6 +35,8 @@ require (
 )
 
 replace (
+	github.com/secure-systems-lab/go-securesystemslib => github.com/secure-systems-lab/go-securesystemslib v0.5.0
+	github.com/sigstore/sigstore => github.com/sigstore/sigstore v1.6.3
```

Comment:
This is to make go "happy" for now.. Ideally we wouldn't have to have those replace, but without it, it fails miserably...

Reply:
should we try bumping chains to latest main ?